{"name": "Невозможно войти в систему, используя только пароль.", "status": "passed", "description": "Проверяется: ошибка == \"Недостаточно данных для входа\" и статус код == 400.", "steps": [{"name": "Регистрация курьера и подготовка json", "status": "passed", "start": 1721097478704, "stop": 1721097479986}, {"name": "Запись пользователя для очистки", "status": "passed", "start": 1721097479986, "stop": 1721097479986}, {"name": "Отправка POST запроса для входа курьера в систему", "status": "passed", "start": 1721097479986, "stop": 1721097481466}, {"name": "Проверка: ошибка == 'Недостаточно данных для входа'", "status": "passed", "start": 1721097481466, "stop": 1721097481466}, {"name": "Проверка: статус код == 400", "status": "passed", "start": 1721097481466, "stop": 1721097481466}], "start": 1721097478703, "stop": 1721097481466, "uuid": "1f6c3660-7a26-4463-a864-ee1f1931f328", "historyId": "c05e44a8eecd0657e1b70110a8ca0254", "testCaseId": "c05e44a8eecd0657e1b70110a8ca0254", "fullName": "tests.test_courier_login.TestCourierLogin#test_login_courier_cannot_login_without_login", "labels": [{"name": "suite", "value": "POST /api/v1/courier/login"}, {"name": "tag", "value": "@pytest.mark.usefixtures('cleanup_users')"}, {"name": "parentSuite", "value": "tests"}, {"name": "subSuite", "value": "TestCourierLogin"}, {"name": "host", "value": "C02DQ0GHMD6R"}, {"name": "thread", "value": "78646-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_courier_login"}]}